pxd.c

```diff
@@ -1500,14 +1530,22 @@ ssize_t pxd_ioc_update_size(struct fuse_conn *fc, struct pxd_update_size *update
 	}
 	(void)get_device(&pxd_dev->dev);
 
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5,11,0)
 	set_capacity(pxd_dev->disk, update_size->size / SECTOR_SIZE);
+#else
+	// set_capacity is sufficient for modifying disk size from 5.11 onwards
+	set_capacity_and_notify(pxd_dev->disk, update_size->size / SECTOR_SIZE);
+#endif
 	spin_unlock(&pxd_dev->lock);
 
+	// set_capacity is sufficient for modifying disk size from 5.11 onwards
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5,11,0)
```

Comment:
this may be needed in 2.8 as this interface is now removed.

Reply:
ok, should test out the resize path before merging.